PYME/DSView/modules/projections.py

```diff
@@ -0,0 +1,45 @@
+
+
+def project(dsviewer, axis, type, crop=False):
+    from PYME.recipes import processing
+    from PYME.DSView import ViewIm3D
+    
+    if crop:
+        from .cropping import crop_2D
+        x0, x1, y0, y1, z0, z1 = dsviewer.do.sorted_selection
+        roi = [[x0, x1 + 1],[y0, y1 +1], [0, dsviewer.image.data.shape[2]]]
+        im = crop_2D(dsviewer.image, roi)
+    else:
+        im = dsviewer.image
+    
+    ViewIm3D(processing.Projection(kind=type, axis=axis).apply_simple(im), parent=dsviewer)
+
+
+def Plug(dsviewer):
+    # type: (PYME.DSView.dsviewer.DSViewFrame) -> object
+    
+    dsviewer.AddMenuItem('Processing>Project>Mean', 'x', lambda e : project(dsviewer, axis=0, type='Mean'))
```

Comment:
I read through this and my first thought was "which project is this for?" Can we change the menu name to `Processing>Projections`? It's slightly more intuitive. Very minor point, but I think a helpful one for users.

Reply:
Seems reasonable (I guess I try to make menu options verbs, but think this would be clearer the other way)